skiplang/prelude/src/skstore/EagerDir.sk

```diff
@@ -502,28 +502,31 @@ class DataMap private {
     }
   }
 
-  fun keysAfter(lastSkippedOpt: ?Boundary<Key>): mutable Iterator<Key> {
+  fun keysAfter(lastSkippedOpt: ?Boundary<K>): mutable Iterator<K> {
     this.data.keysAfter(lastSkippedOpt)
   }
 
   fun getHeight(): Int {
     this.data.getHeight()
   }
 
-  fun remove(k: Key): this {
+  fun remove(k: K): this {
     !this.data = this.data.remove(k);
     !this.tombs = this.tombs.remove(k);
     this
   }
 }
 
-class FixedDataMap{data: FixedData<Array<File>>, tombs: FixedDir<Path>} {
+class FixedDataMap{
```

Comment:
What's the blocker on this (and EagerDir) also getting a type parameter `K`?  in practice these should always be keyed on some more-precise-than-`Key` type, right?

Or, is there no blocker and you're just planning to get to those in part 2?